### PR TITLE
Ensure `ZPFX/man/man{1..9}` exist

### LINKED ISSUE #8: MANPATH doesn't have ~/.zinit/polaris/man, also ~/.zinit/polaris/man doesn't exist
**Describe the bug**
ZPFX=~/.zinit/polaris
MANPATH doesn't have $ZPFX/man, also $ZPFX/man doesn't exist.

**To Reproduce**
```bash
# requires the zdharma-continuum/z-a-bin-gem-node annex
# i'm also using ryaminal/z-a-meta-plugins to bring in the annexes(including the one above), but I don't anticipate this being a problem.
zinit wait'0b' lucid for \
  id-as'github-cli' from"gh-r" sbin'usr/bin/gh' atclone'ln -sf $PWD/usr/share/man/man1/* $ZPFX/man/man1' atpull'%atclone' cli/cli \
  ;
```

**Expected behavior**
Used to be that a $ZPFX/man directory was created and added to the MANPATH directory, this is the expected behavior.

**Additional context**
Mostly I'm just curious if anyone is familiar with why this wouldn't work? Are others seeing this directory existing and an entry in MANPATH? I've tried to look through the code but nothing is indicating the problem.

### PR DESCRIPTION
**Pull request by pschmitt**
Fixes #8 

This PR automates the creation of `${ZINIT[MAN_DIR]:-ZPFX/man/man}{1..9}`.
Overriding `$ZINIT[MAN_DIR]` will make zinit create the directory, but it may not be added automatically to `manpath`.

See #8 for more details.

### CODE REVIEW

File: zinit.zsh [hunk 1]
```diff
@@ -77,6 +77,7 @@ pullopts|debug|null|binary|make|nocompile|notify|reset"
 typeset -g ZPFX
 : ${ZPFX:=${ZINIT[HOME_DIR]}/polaris}
 : ${ZINIT[ALIASES_OPT]::=${${options[aliases]:#off}:+1}}
+: ${ZINIT[MAN_DIR]:=${ZPFX}/man/man1}
```
Comment (fugkco) [inline zinit.zsh:80]: what about other man dirs? `man{1...9}` are all valid man directories. See `man man.1` for more info.

Comment (pschmitt) [reply zinit.zsh:80]: Fair point. I don't have anything in my current config that uses anything other than man1. But yeah, valid. Will adjust.

Comment (pschmitt) [reply zinit.zsh]: Done da91b76

**Comment by vladdoster** [comment]
Can you update the [CHANGELOG.md](https://github.com/zdharma-continuum/zinit/blob/master/doc/CHANGELOG.md)?

**Comment by pschmitt** [comment]
> Can you update the CHANGELOG.md?

I can, I can. If feels weird to not announce the new home in there though (out of scope for this PR)
I'll do this in a separate PR.

**Pull request merged**